widget/topPanelWidget.js

```diff
@@ -20,18 +20,16 @@ var TopPanel = GObject.registerClass(
             this.taskBar = new TaskBar(superWorkspace);
             this._leftContainer.add_child(this.taskBar);
 
-            let iconContainer = new St.Bin({
-                child: new St.Icon({
-                    gicon: Gio.icon_new_for_string(
-                        `${Me.path}/assets/icons/plus-symbolic.svg`
-                    ),
-                    style_class: 'mat-panel-button-icon'
-                }),
-                style_class: 'mat-panel-button'
+            let iconContainer = new St.Icon({
+                gicon: Gio.icon_new_for_string(
+                    `${Me.path}/assets/icons/plus-symbolic.svg`
+                ),
+                style_class: 'mat-panel-button-icon'
```

Comment:
I updated `iconContainer` to match up more closely with `tilingIcon/tilingButton` in terms of what widgets they are built from.